client/src/components/profile.ts

```diff
@@ -88,6 +88,7 @@ export interface SSHProfile extends BaseProfile {
   saspath: string;
   port: number;
   username: string;
+  identityFile: string;
```

Comment:
Can this be made as an optional field? I think having it as optional is more on brand for the overall approaches we recommend for this in the markdown changes. It should also clean up the tests a bit so that the "original" tests dont have to be burdened with keeping up with default values. In my mind we would write a separate set of tests that isolates how this new member gets passed around etc.

Reply:
With apologies, this is the first time I've ever used either typescript, or VS Code. In my head this is already "_optional_", in that when the profile is created, you can leave it blank.

I intuit you mean "_optional_" like "_the TS type system affords making it explicit that this field is optional_", but it's not obvious to me how to implement that in a way that aligns with the rest of the codebase. I would appreciate pointers to the right path.